app/src/androidTest/groovy/com/macoscope/ketchuplunch/graybox/LunchMenuWithStubsSpec.groovy

```diff
@@ -0,0 +1,43 @@
+package com.macoscope.ketchuplunch.graybox
+
+import android.content.Intent
+import android.support.test.espresso.assertion.ViewAssertions
+import android.support.test.rule.ActivityTestRule
+import com.macoscope.ketchuplunch.R
+import com.macoscope.ketchuplunch.di.LunchModule
+import com.macoscope.ketchuplunch.model.ScriptClient
+import com.macoscope.ketchuplunch.view.lunch.LunchActivity
+import org.junit.Rule
+import org.mockito.Mockito
+import spock.lang.Specification
+
+import static android.support.test.espresso.Espresso.onView
+import static android.support.test.espresso.matcher.ViewMatchers.withText
+import static com.macoscope.ketchuplunch.RecyclerViewMatcher.withRecyclerView
+import static org.mockito.BDDMockito.given
+import static org.mockito.Matchers.isA
+
+class LunchMenuWithStubsSpec extends Specification {
+
+    @Rule
+    ActivityTestRule<LunchActivity> loginActivityRule = new ActivityTestRule(LunchActivity, true, false)
+
+    ScriptClient scriptClientMock = Mockito.mock(ScriptClient)
+
+    def setup() {
+        LunchModule.scriptClient = scriptClientMock
+    }
+
+    def "displays menu for Monday"() {
+        given:
+            ArrayList<Map<String, Object>> apiResult = new ArrayList<Map<String, Object>>()
+            apiResult.add(["name": "Onion Soup", "count": 1 as BigDecimal, "totalCount": 3 as BigDecimal])
```

Comment:
checking 2 elements is redundant, I have to display at least 2 to make use of RecyclerViewMatcher, will change name of first element